src/lib.rs

```diff
@@ -608,9 +602,10 @@ mod tests {
         #[test]
         fn vector_of_non_strings() {
             let context = Context::test();
-            let config: Vec<LogCommand> = vec![LogCommand];
+            let log_commands: Vec<LogCommand> = vec![LogCommand];
             let StdoutTrimmed(stdout) =
-                crate::run_result_with_context!(context.clone(), config, %"echo foo").unwrap();
+                run_result_with_context(context.clone(), (log_commands, Split("echo foo")))
+                    .unwrap();
```

Comment:
A lot of these tests changed from using the macro interface to using a tuple based interface through `run_result_with_context` and `run_result_with_context_unit`.